### DISCUSSION TITLE
Allow 'Disable left-click' for the placeholder toggle

### DISCUSSION THREAD

**Discussion by Memebeams** [runelite/runelite#14943] [Ideas]
Currently, the bank plugin allows you to avoid accidental deposits by disabling left-click on deposit buttons:
![image](https://user-images.githubusercontent.com/5132037/168105037-60824a47-35d2-4a21-a7e4-7356082c6efb.png)

Sometimes, I will remove items from my bank without realizing that the placeholder button was turned off and mess up my bank arrangement. This button would make it harder for dingbats like me to turn placeholders off

**Comment by Nightfirecat** [comment]
I've created #14961 for implementing this enhancement.
